RestAPIGateway/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM openjdk:8-jre
+
+ARG VCS_REF
+ARG BUILD_DATE
+ARG port
+
+ENV expose_port=$port
+
+LABEL org.label-schema.vcs-ref=$VCS_REF \
+      org.label-schema.vendor="Microsoft" \
+      org.label-schema.name="pumrp-api" \
```

Comment:
For consistency, let's add org.label-schema.url and org.label-schema.vcs-url per https://github.com/Microsoft/PartsUnlimitedMRPmicro/commit/399ceaf97065b3748703c3ef0a70fcb5426aacb5

Reply:
done